corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py

```diff
@@ -66,14 +66,16 @@ def handle(self, **options):
             logger.info('migrating %s: %s apps', domain, len(app_ids))
             for app_id in app_ids:
                 try:
-                    if self.fix_user_props:
-                        self.fix_user_properties(app_id)
+                    app = get_app(domain, app_id)
+                    if app.doc_type == "Application":
+                        if self.fix_user_props:
+                            self.fix_user_properties(app)
+                        else:
+                            self.migrate_app(app)
                     else:
-                        self.migrate_app(app_id)
-                except SkipApp as err:
+                        logger.info("Skipping %s/%s because it is a %s", domain, app_id, app.doc_type)
+                except Exception as err:
                     logger.error("skipping app %s/%s: %s", domain, app_id, err)
-                except Exception:
-                    logger.exception("skipping app %s/%s", domain, app_id)
```

Comment:
I would prefer to bring back this `except` clause instead of the one above it. That is, I think it should use `logger.exception`, which logs the stack trace, and don't put the error name in the message since it's in the stack trace.